### PR TITLE
Adding an option to not allow all cluster:monitor actions

### PR DESCRIPTION
**Pull request by jehuty0shift**
Hello, First of all thanks for porting guard to ES 1.7 by creating armor. I'm now trying to extend the armor functionalities. This pull request has for goal to not allow all cluster:monitor actions if they are not forbidden explicitly. For example we want to allow only cluster:monitor/health and cluster:monitor/nodes to allow kibana to work but we don't want to give the users the ability to see others informations given by the _cat endpoint. 
Basically i added an option to disable the special treatment of cluster:monitor actions. the default value of the option will imply the same behavior as before. I also added a small test. If you have time could you review my code ?

### CODE REVIEW

**Comment by jmaitrehenry** [comment]
Hi @jehuty0shift!

Thanks for your pull request, I will try to review it this week after closing the 2.0 support.

Is it possible to update the README or create a new document file about it? I want to update the documentation and how to configure the module, and set rules. I have the `update_doc` branch for that.

**Comment by splitice** [comment]
+1
